app/services/clients/products_client.js

```diff
@@ -96,9 +96,29 @@ function getProductsByGatewayAccountId (gatewayAccountId) {
   }).then(products => products.map(product => new Product(product)))
 }
 
+/**
+ * @param {String} gatewayAccountId: the external id of the product you wish to retrieve
+ * @returns {Promise<Product>}
+ */
+function updateServiceNameOfProductsByGatewayAccountId (gatewayAccountId, serviceName) {
+  return baseClient.patch({
+    headers,
+    baseUrl,
+    url: `/gateway-account/${gatewayAccountId}`,
+    json: true,
+    body: {
+      op: 'update',
```

Comment:
op:replace, otherwise it will be rejected as `Operation update is not valid for path service_name`

Reply:
Ahh I see. Sorted